### ISSUE TITLE
Compilation issue

### ISSUE THREAD

**Issue by liberodark** [ValveSoftware/wine#120]
Hi,

I know you don't support a compilation out of proton build system.
But have fixe issue on 6.3-5 but 6.3-6 have new compilation issue.
And my OS don't support your build system.
@aeikum If you have any idea how to fix ?

```
dlls/user32/focus.c: In function ‘GetForegroundWindow’:
dlls/user32/focus.c:372:78: error: ‘const struct get_thread_input_reply’ has no member named ‘foreground’
  372 |         if (!wine_server_call_err( req )) ret = wine_server_ptr_handle( reply->foreground );
      |                                                                              ^~
make[2]: *** [Makefile:125538: dlls/user32/focus.o] Error 1
make[2]: *** Waiting for unfinished jobs....
dlls/user32/input.c: In function ‘GetCursorInfo’:
dlls/user32/input.c:293:57: error: ‘const struct get_thread_input_reply’ has no member named ‘cursor’
  293 |             pci->hCursor = wine_server_ptr_handle( reply->cursor );
      |                                                         ^~
dlls/user32/input.c:294:32: error: ‘const struct get_thread_input_reply’ has no member named ‘show_count’
  294 |             pci->flags = (reply->show_count >= 0) ? CURSOR_SHOWING : 0;
      |                                ^~
make[2]: *** [Makefile:125598: dlls/user32/input.o] Error 1
dlls/user32/message.c: In function ‘peek_message’:
dlls/user32/message.c:2743:50: error: ‘const struct get_message_reply’ has no member named ‘active_hooks’
 2743 |                 thread_info->active_hooks = reply->active_hooks;
      |                                                  ^~
make[2]: *** [Makefile:125696: dlls/user32/message.o] Error 1
```

Best Regards

**Comment by aeikum** [comment]
@liberodark Thanks for reporting, I think I see the issue.

**Comment by aeikum** [comment]
As a workaround, you can run "./tools/make_requests" in the wine subdir, which I think should fix this. But I'll work on fixing it properly.

**Comment by aeikum** [comment]
@liberodark I've pushed a new commit to the -next branch to fix this: b12968e2d82 . Please update and give it a try.

**Comment by aeikum** [comment]
Hi again @liberodark . After some internal discussion, it seems this actually isn't an error. The Proton build system automatically regenerates those files (see the post-wine-source target: https://github.com/ValveSoftware/Proton/blob/82ae47758ff9305675624589dfb7bea5928ae956/build/makefile_base.mak#L670 ). This is much more convenient for us during development, so we don't have to track those generated file changes. So I will probably drop that new commit in the final release of 6.3-6 after all.

Please update your build system to call `autoreconf -f` and `./tools/make_requests` before building Wine.

**Comment by liberodark** [comment]
Hi @aeikum your fix work b12968e2d822d9eb621c67cadfd53e5dcd2ef11e
Thank you so much for that.
We have been using wine-proton for a long time and hope to be able to keep it in our OS. 
If you can keep the compatibility that would be really great.
In case if you want to see our MK : 
https://github.com/batocera-linux/batocera.linux/blob/master/package/batocera/emulators/wine/wine-proton/wine-proton.mk

Best Regards

**Comment by liberodark** [comment]
Ok have added `autoreconf -f` and `./tools/make_requests ` now is ok.
@aeikum 

Best Regards
